### ISSUE TITLE
'npm run firefox' throwing error on archlinux

### ISSUE THREAD

**Issue by jakehm** [devtools-html/debugger.html#842]
Running on archlinux.  'npm start' is working.

```
[jake@localhost debugger.html]$ npm run firefox

> debugger.html@0.0.5 firefox /home/jake/code/debugger.html
> node bin/firefox-driver --start

/home/jake/code/debugger.html/node_modules/selenium-webdriver/lib/promise.js:2291
        throw error;
        ^

WebDriverError: Error loading page
    at WebDriverError (/home/jake/code/debugger.html/node_modules/selenium-webdriver/lib/error.js:27:5)
    at Object.throwDecodedError (/home/jake/code/debugger.html/node_modules/selenium-webdriver/lib/error.js:477:11)
    at parseHttpResponse (/home/jake/code/debugger.html/node_modules/selenium-webdriver/lib/http.js:388:15)
    at doSend.then.response (/home/jake/code/debugger.html/node_modules/selenium-webdriver/lib/http.js:330:11)
    at process._tickCallback (internal/process/next_tick.js:103:7)
From: Task: WebDriver.navigate().to(http://localhost:7999/todomvc)
    at Driver.schedule (/home/jake/code/debugger.html/node_modules/selenium-webdriver/lib/webdriver.js:414:17)
    at Navigation.to (/home/jake/code/debugger.html/node_modules/selenium-webdriver/lib/webdriver.js:1042:25)
    at Driver.get (/home/jake/code/debugger.html/node_modules/selenium-webdriver/lib/webdriver.js:832:28)
    at Object.<anonymous> (/home/jake/code/debugger.html/bin/firefox-driver.js:51:10)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.runMain (module.js:590:10)

```

**Comment by jlongster** [comment]
We've seen a few people report issues from linux (I think specifically arch). I'm not sure what the problem is; I'm not very familiar with WebDriver. In the short term, it's not very hard to manually configure Firefox and tell it to listen on a debug port: https://github.com/devtools-html/debugger.html/blob/master/docs/remotely-debuggable-browsers.md#firefox

**Comment by jasonLaster** [comment]
Closing this issue as it's likely stale. We've made some improvements to the driver (upgrading to selenium 3). The primary issue now is https://github.com/devtools-html/debugger.html/issues/906, where the driver no longer sets the debugger server

**Comment by jasonLaster** [comment]
This is not an issue with linux, it's an issue with the examples server not being started, which was just fixed in #1201
